include/openenclave/host.h

```diff
@@ -157,13 +150,8 @@ oe_result_t oe_create_enclave(
     const char* path,
     oe_enclave_type_t type,
     uint32_t flags,
-#ifdef OE_CONTEXT_SWITCHLESS_EXPERIMENTAL_FEATURE
     const oe_enclave_config_t* configs,
```

Comment:
For consistency with the other changes, I believe this should be something like:
oe_enclave_setting_t* settings,